modules/local/fusionreport/detect/main.nf

```diff
@@ -4,7 +4,7 @@ process FUSIONREPORT {
 
     // Note: 2.7X indices incompatible with AWS iGenomes.
     conda "bioconda::star=2.7.9a"
-    container "docker.io/clinicalgenomics/fusion-report:2.1.5p2"
+    container "docker.io/clinicalgenomics/fusion-report:2.1.5p2.1"
```

Comment:
Did not notice before that you used this container, has Martin stopped maintaining `fusion-report`?

Reply:
Martin is doing a rewrite. The fork was to make testing easier for me, permission-wise. @matq007 do you want to integrate the fork?